tests/unittests/tests-core/tests-core-xfa.c

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (C) 2018 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdint.h>
+#include "xfa.h"
```

Comment:
```suggestion
#include "bitarithm.h"
#include "xfa.h"
```

Please squash right away.